config/config.yml

```diff
@@ -0,0 +1,50 @@
+# Important for understanding DANE configs: https://github.com/CLARIAH/DANE/blob/main/DANE/config.py
+# To read more about the configuration: https://github.com/beeldengeluid/dane-example-worker/wiki/Config
+
+# Important note:
+# FIRST the home dir config is applied (~/.DANE/config.yml),
+# THEN the base_config.yml will overwrite anything,
+# THEN the local config.yml
+
+# Note: For local testing, copy this file to config.yml (in main dir of this repo)
+# Or export DANE_HOME=./config to point DANE to this file
+
+RABBITMQ:
+    HOST: dane-rabbitmq-api.default.svc.cluster.local
+    PORT: 5672
+    EXCHANGE: DANE-exchange
+    RESPONSE_QUEUE: DANE-response-queue
+    USER: guest # change this for production mode
+    PASSWORD: guest # change this for production mode
+ELASTICSEARCH:
+    HOST:
+        - elasticsearch
+    PORT: 9200
+    USER: '' # change this for production mode
+    PASSWORD: '' # change this for production mode
+    SCHEME: http
+    INDEX: dane-index-k8s
+FILE_SYSTEM:
+    BASE_MOUNT: /data # /data when running in a container
+    INPUT_DIR: input-files
+    OUTPUT_DIR: output-files
+INPUT:
+    TEST_INPUT_PATH: testsource__testcarrier/
+    S3_ENDPOINT_URL: https://s3-host
+    MODEL: s3://bucket/model
+    DELETE_ON_COMPLETION: False
+OUTPUT:
+    DELETE_ON_COMPLETION: True
+    TRANSFER_ON_COMPLETION: True
+    S3_ENDPOINT_URL: https://s3-host
+    S3_BUCKET: bucket-name
+    S3_FOLDER_IN_BUCKET: folder  # folder within the bucket
+MM_EX_EMO_SETTINGS:
+    DATA_DIR: /Structured_input_data
```

Comment:
Make use of the FILESYSTEM variables, rather than hardcode these locations. Structured_input_data might as well land in /data, and you can add a MODEL_DIR under FILE_SYSTEM 
This way, you only need to change one thing when running locally instead of containerized